docker-compose.yml

```diff
@@ -273,6 +279,65 @@ services:
       SENTRY_DSN: http://user:password@error-handler.dev.gov.uk/123
     ports: []
 
+  collections-publisher: &collections-publisher
+      build: apps/collections-publisher
+      depends_on:
+        - publishing-api
+        - mysql
+        - rummager
+        - collections-publisher-worker
+        - diet-error-handler
+      environment:
+        SENTRY_CURRENT_ENV: collections-publisher
+        SENTRY_DSN: http://user:password@error-handler.dev.gov.uk/123
+        VIRTUAL_HOST: collections-publisher.dev.gov.uk
+      links:
+        - mongo
+        - mysql
+        - redis
+        - nginx-proxy:rummager.dev.gov.uk
+        - nginx-proxy:publishing-api.dev.gov.uk
+        - nginx-proxy:error-handler.dev.gov.uk
+      ports:
+        - "33071:3071"
+      volumes:
+        - ./apps/collections-publisher/log:/app/log
+
+  collections-publisher-worker:
+    << : *collections-publisher
+    command: bundle exec sidekiq -C ./config/sidekiq.yml
+    depends_on:
+      - publishing-api
+      - diet-error-handler
+    environment:
+      SENTRY_CURRENT_ENV: collections-publisher-worker
+      SENTRY_DSN: http://user:password@error-handler.dev.gov.uk/123
+    ports: []
+
+  collections: &collections
+      build: apps/collections
+      depends_on:
+        - content-store
+        - static
+        - rummager
+        - diet-error-handler
+      environment:
+        PLEK_SERVICE_SEARCH_URI: http://rummager.dev.gov.uk
+        SENTRY_CURRENT_ENV: collections
+        SENTRY_DSN: http://user:password@error-handler.dev.gov.uk/123
+        VIRTUAL_HOST: collections.dev.gov.uk
+      links:
+        - mongo
```

Comment:
is this intended?

Reply:
Agreed, collections shouldn't need a link to `mongo`.